src/features/client/components/Header/index.jsx

```diff
@@ -38,14 +40,17 @@ function Header() {
           <Link as={ReachLink} to="/">
             <Image boxSize="30px" src={logoUrl} />
           </Link>
-          <ButtonGroup gap="20px">
-            <Button onClick={onOpen} variant="ghost">
-              Log In
-            </Button>
-            <Button onClick={onOpen} colorScheme="blue" color="white">
-              Sign up
-            </Button>
-          </ButtonGroup>
+          {!basket && (
```

Comment:
Leave the login and register button, since guest user can also order (show login, register and basket)